app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<p>In this page all the users will be displayed.</p>
+<div id="users-index">
+  <ul>
+    <% @users.each do |user| %>
+      <li class="user-card">
+        <img src="<%= user.photo %>" alt="User photo">
+        <div class="user-info">
+          <h2><%= user.name %></h2>
```

Comment:
- [ ] It can be good if the user names are links that users can click so that they are taken to the show page. This is important because this is the only way users can view posts and comments.